packages/core/src/utils/errors.ts

```diff
@@ -6,6 +6,7 @@ export enum CivilErrors {
   NoUnlockedAccount = "NO_UNLOCKED_ETHEREUM_ACCOUNT",
   NoPrivileges = "NOT_ENOUGH_PRIVILEGES_FOR_ACTION",
   EvmException = "EVM_EXCEPTION_OCCURED",
+  NoChallenge = "NO_CHALLENGE_FOUND_FOR_LISTING",
```

Comment:
The goal for those errors is to have errors that can be caught in the UI and be handled and shown to the user with a nice explanation. I wonder if this one's too granular. How about `InvalidParameters` or similar?

Reply:
I don't think InvalidParameters really makes sense and wouldn't be particularly helpful (user might think the listingAddress they inputted is wrong). I don't think NoChallenge is too granular, we can discuss changing it later if we think it is though.